internal/suites/const.go

```diff
@@ -1,10 +1,25 @@
 package suites
 
-import "fmt"
+import (
+	"fmt"
+	"os"
+)
+
+// GetLoginBaseURL returns the URL of the login portal and the path prefix if specified.
+func GetLoginBaseURL() string {
```

Comment:
@clems4ever is this okay to reside here or would you rather I move it to something like `hooks.go`?